items/PET_SKIN_HORSE_ZOMBIE.json

```diff
@@ -0,0 +1,30 @@
+{
+  "itemid": "minecraft:skull",
+  "displayname": "§f§f§5Zombie Skeleton Horse Skin",
+  "nbttag": "{overrideMeta:1b,HideFlags:254,SkullOwner:{Id:\"76e06cd3-376f-3cfa-9431-1c2b67de42e9\",Properties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTYwNTQwMjg1NDk2NCwKICAicHJvZmlsZUlkIiA6ICJmMzA1ZjA5NDI0NTg0ZjU4YmEyYjY0ZjAyZDcyNDYyYyIsCiAgInByb2ZpbGVOYW1lIiA6ICJqcm9ja2EzMyIsCiAgInNpZ25hdHVyZVJlcXVpcmVkIiA6IHRydWUsCiAgInRleHR1cmVzIiA6IHsKICAgICJTS0lOIiA6IHsKICAgICAgInVybCIgOiAiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS81NzgyMTFlMWI0ZDk5ZDFjN2JmZGE0ODM4ZTQ4ZmM4ODRjM2VhZTM3NmY1OGQ5MzJiYzJmNzhiMGE5MTlmOGU3IgogICAgfQogIH0KfQ==\"}]}},display:{Lore:[0:\"§8Consumed on use\",1:\"\",2:\"§7§7Pet skins change the look and\",3:\"§7particle trail of your pet but\",4:\"§7only one skin can be active at a\",5:\"§7time!\",6:\"\",7:\"§7§7This skin can only be applied\",8:\"§7to Skeleton Horse pets.\",9:\"\",10:\"§7§eRight click on your pet to\",11:\"§eapply this skin!\",12:\"\",13:\"§9§lRARE COSMETIC\"],Name:\"§f§f§5Zombie Skeleton Horse Skin\"},ExtraAttributes:{originTag:\"FIRE_SALE\",id:\"PET_SKIN_HORSE_ZOMBIE\",uuid:\"ceeb8fab-512d-40a8-8835-a3a1ff4c7d27\",timestamp:\"1/29/21 12:00 PM\"},AttributeModifiers:[]}",
```

Comment:
why is this not blue? it still looks like it is valid

Reply:
I checked it again and it's an epic cosmetic with a purple name so It's not supposed to be blue but in the nbt tag I accidentaly left 'RARE COSMETIC' so I will fix that now